systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlConfigurationST.java

```diff
@@ -63,38 +65,87 @@ public class CruiseControlConfigurationST extends AbstractST {
 
     private final String namespace = testSuiteNamespaceManager.getMapOfAdditionalNamespaces().get(CruiseControlConfigurationST.class.getSimpleName()).stream().findFirst().get();
 
+    private static Double removeKibSuffix(String s) {
+        return Double.valueOf(s.substring(0, s.length() - 5));
+    }
+
```

Comment:
This method should be in `StUtils` or something like that FMPOV.

Reply:
Since this is very specific to this test method and not used in any other classes I really think we should leave it here but if you feel strongly about the move I will move it!

Reply:
Yeah, I know. The thing is that when someone will write, for example, similar test, he will create new method (if he wouldn't know that there is similar method already "implemented") -> for these things we are adding them to `*Utils` classes. We have also `CruiseControlUtils`, if you don't want to add it to generic `StUtils`.